src/Model/Profile.php

```diff
@@ -643,27 +643,29 @@ public static function getEventsReminderHTML()
 		$bd_format = L10n::t('g A l F d'); // 8 AM Friday January 18
 		$classtoday = '';
 
+		$self = dba::selectFirst('contact', ['id'], ['uid' => local_user(), 'self' => true]);
+
 		$s = dba::p(
 			"SELECT *
```

Comment:
A ```SELECT *``` with a ```JOIN``` would add all the fields from the item table as well, I think. Some ```SELECT event.*``` would be better.

Reply:
Well spotted.